### PR TITLE
[master] Fix for incorrect chain ID when it is not 1

### PR DESCRIPTION
**Pull request by n-hutton**

### CODE REVIEW

File: src/libData/AccountData/AccountStoreSC.tpp [hunk 1]
```diff
@@ -447,7 +447,9 @@ bool AccountStoreSC<MAP>::UpdateAccounts(const uint64_t& blockNum,
       // Check if the sender has enough balance to pay gasDeposit
       if (fromAccount->GetBalance() < gasDeposit + transaction.GetAmount()) {
         LOG_GENERAL(WARNING,
-                    "The account doesn't have enough gas to create a contract");
+                    "The account doesn't have enough gas to create a contract. Bal: "
```
Comment (valeryz) [inline src/libData/AccountData/AccountStoreSC.tpp:450]: This is useful, but various stability reasons we like to keep the changes to AccountStoreSC.tpp to the bare minimum.    Makes sense to apply this change to Steve's PR: https://github.com/Zilliqa/Zilliqa/pull/2895

File: src/libServer/LookupServer.cpp [hunk 2]
```diff
@@ -1324,6 +1324,9 @@ Json::Value LookupServer::GetEthTransactionReceipt(const std::string& txnhash) {
 
     std::string blockHash = "";
 
+    std::cout << "Getting TX receipt for: " << txnhash << std::endl;
```
Comment (valeryz) [inline src/libServer/LookupServer.cpp:1327]: Should we log to stdout, or use LOG_GENERAL?

Comment (n-hutton) [reply src/libServer/LookupServer.cpp:1327]: This is just debug to assist saeed and me (only a draft pr now for collaboration)

File: src/libData/AccountData/AccountStoreSC.tpp (hunk 1 above)
Comment (n-hutton) [reply src/libData/AccountData/AccountStoreSC.tpp:450]: steves PR has been merged already - do you want me to exclude this change from master?

**Pull request closed without merging**